### PR TITLE
create tests for navigations endpoint

### PR DESCRIPTION
**Pull request by m-alama**
Create a positive and negative tests for navigations endpoint

### CODE REVIEW

File: tests/api/b2b/glue/navigations_endpoints/positive.robot [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][nodes]    url
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][nodes]    cssClass
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][nodes]    validFrom
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][nodes]    validTo
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][nodes]    children
+    And Response Body parameter should have datatype:    [data][attributes][nodes][0][nodeType]    str
+    And Response Body parameter should have datatype:    [data][attributes][nodes][0][title]    str
+    And Response Body parameter should have datatype:    [data][attributes][nodes][0][url]    str
+    And Response Body parameter should have datatype:    [data][attributes][nodes][0][children]    list
+    And Response body has correct self link internal
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/navigations_endpoints/positive.robot:32]: please add checks that at least one of the nodes has actual values in 
nodeType
isActive
url
cssClass
(can check that they are not empty, no need to check literal values)

File: tests/api/b2b/glue/navigations_endpoints/positive.robot [hunk 2]
```diff
@@ -0,0 +1,33 @@
+*** Settings ***
+Suite Setup    SuiteSetup
+Test Setup    TestSetup
+Resource    ../../../../../resources/common/common_api.robot
+Default Tags    glue
+
+*** Test Cases ***
+
+
+Get_navigation_tree_using_valid_navigation_key
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/navigations_endpoints/positive.robot:10]: please add another test with the node that has children and check that children have correct properties and some important values are not empty.

Keywords for the checks of the elements of a nested arrays will be merged to master today if you need them, but if they are not there just pick one child and check it.

File: tests/api/b2b/glue/navigations_endpoints/positive.robot (hunk 1 above)
Comment (m-alama) [reply tests/api/b2b/glue/navigations_endpoints/positive.robot:40]: Sometimes url can be null

File: tests/api/b2b/glue/navigations_endpoints/positive.robot (hunk 2 above)
Comment (m-alama) [reply tests/api/b2b/glue/navigations_endpoints/positive.robot:10]: children keys checked that exists

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**